app/service_invite/rest.py

```diff
@@ -48,9 +49,28 @@ def _create_service_invite(invited_user, invite_link_host):
         current_app.config["SECRET_KEY"],
         current_app.config["DANGEROUS_SALT"],
     )
+
+    # The raw permissions are in the form "a,b,c,d"
+    # but need to be in the form ["a", "b", "c", "d"]
+    data = {}
+    permissions = invited_user.permissions
+    permissions = permissions.split(",")
+    permission_list = []
+    for permission in permissions:
+        permission_list.append(f"{permission}")
```

Comment:
Are the permissions not already in a string format, or would this work instead?

```suggestion
    permission_list = permissions.split(",")
```

Reply:
And if the explicit string conversion is needed, then this?

```suggestion
    permission_list = [ str(permission) for permission in permissions.split(",") ]
```